src/pages/index.js

```diff
@@ -0,0 +1,348 @@
+import {
+    enableValidation,
+    settings,
+    resetValidation,
+    disabledButton,
+} from "../scripts/validation.js";
+import "./index.css";
+import Api from "../utils/Api.js";
+import { setbuttonText } from "../utils/helpers.js";
+// import { set } from "core-js/core/dict";
+
+// const initialCards = [
+//     {
+//         name: "Val Thorens",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
+//     },
+//     {
+//         name: "Restaurant terrace",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//     },
+//     {
+//         name: "An outdoor cafe",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//     },
+//     {
+//         name: "A very long bridge, over the forest and through the trees",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//     },
+//     {
+//         name: "Tunnel with morning light",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//     },
+//     {
+//         name: "Mountain house",
+//         link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//     },
+// ];
+
+const api = new Api({
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+        authorization: "5644952e-2d27-433e-aa79-12a5c9f211e5",
+        "Content-Type": "application/json",
+    },
+});
+
+api.getAppInfo()
+    .then(([cards, userInfo]) => {
+        cards.forEach((item) => {
+            const cardElement = getCardElement(item);
+            cardsList.prepend(cardElement);
+        });
+        profileName.textContent = userInfo.name;
+        profileDescription.textContent = userInfo.about;
+        const profileAvatar = document.querySelector(".profile__avatar");
```

Comment:
`        const profileAvatar = document.querySelector(".profile__avatar");
`

You've already found the avatar in line 65 as `avatarModalBtn`. Please, use it here

Reply:
* Another remark hasn't been corrected https://github.com/Luidelson/se_project_spots/pull/1#discussion_r2094621833